gems/recaptcha/.gitignore

```diff
@@ -1,10 +0,0 @@
-rdoc
```

Comment:
Not relevant to this file but I removed `gems/recaptcha` since we're not longer customizing it. *Yay!*